frontend/app/routes/public/apply/$id/child/review-adult-information.tsx

```diff
@@ -74,7 +74,7 @@ export async function loader({ context: { configProvider, serviceProvider, sessi
   const countryHome = serviceProvider.getCountryService().getCountryById(state.contactInformation.homeCountry);
 
   // Getting CommunicationPreference by Id
-  const communicationPreference = serviceProvider.getPreferredCommunicationMethodService().findById(state.communicationPreferences.preferredMethod);
+  const communicationPreference = serviceProvider.getPreferredCommunicationMethodService().findPreferredCommunicationMethodsById(state.communicationPreferences.preferredMethod);
   invariant(communicationPreference, `Unexpected communication preference: ${state.communicationPreferences.preferredMethod}`);
```

Comment:
**Suggestion:** Having the service expose a `get(..)` instead of a `find(..)`, based on our conventions, would mean that the service would throw if no `communicationPreference` was found rather than returning `null`.

Throwing early in the service would eliminate the need for all the `invariant` checks afterwards where `findPreferredCommunicationMethodsById(..)` is currently called.